python/tank_vendor/shotgun_base/utils.py

```diff
@@ -141,6 +141,29 @@ def sanitize_path(path, separator=os.path.sep):
     return local_path
 
 
+def ensure_file_exists(path, permissions=0666):
```

Comment:
Sure thing. I pretty much copied what was in the hook into here and followed the `ensure_folder_exists()` convention. But `touch_file()` works for me. :)